manifests/w/win/capture/audio/2.2.2/win.capture.audio.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: win.capture.audio

```

```suggestion
PackageIdentifier: bozbez.WinCaptureAudio.beta
```
This better aligns with `publisher.package.channel`